test_minival_locally.sh

```diff
@@ -20,11 +20,11 @@ esac
 done
 
 
-docker run -v $(pwd)/gibson_challenge_data_2021:/opt/iGibson/gibson2/data \
+docker run -v $(pwd)/igibson.key:/opt/iGibson/igibson/data/igibson.key -v $(pwd)/ig_dataset:/opt/iGibson/igibson/data/ig_dataset \
```

Comment:
this shouldn't be necessary anymore, we can just have them move igibson.key under data and bindmount  the entire data folder no? Unless we still want the auto-download to happen in docker.

Reply:
Yeah I still added the auto-download in docker. It's consistent with the rest of our documentation.

Essentially ig_assets is always auto-downloaded (no need to sign anything).
ig_dataset and igibson.key are acquired from signing the license agreement.

Reply:
Yeah no I get that, it just doesn't seem easier to me lol

Reply:
I think it's is conceptually easier for the users. They just need to 1) sign the license agreement, 2) download `igibson.key` and `ig_dataset` from that google form, and 3) run `test_minival_locally.sh`. 

Otherwise, the users need to manually `wget https://storage.googleapis.com/gibson_scenes/assets_igibson.tar.gz`, which doesn't match any of our existing documentation.

Reply:
your call :) It's fine then. I think it's easier to add a step to the documentation which is 
```
mkdir data && cd data
wget https://storage.googleapis.com/gibson_scenes/assets_igibson.tar.gz
tar -xzf assets_igibson.tar.gz
unzip ig_dataset.zip
cp ~/downloads/igibson.key .
```

and then have them bindmount that directory

It's totally your call though, what you wrote is fine :)

Reply:
Yeah, I got what you saying. In my view, they are similarly easy. It's just that we have never exposed `https://storage.googleapis.com/gibson_scenes/assets_igibson.tar.gz` to the users in our existing documentation (users always just do `python -m igibson.utils.assets_utils --download_assets`). I would love to keep it that way.